website/docs/reactr/language-support.md

```diff
@@ -2,8 +2,7 @@
 
 Reactr supports a growing number of languages for writing [Runnables](../atmo/concepts/runnables.md) in. Projects in the Suborbital ecosystem such as [Atmo](../atmo/) or [Sat](../sat/) use the Reactr runtime to execute these WebAssembly, allowing them to interface with the runtime through the [Runnable interface](../atmo/runnable-api/introduction#the-runnable-interface). The supported languages may vary across Reactr releases, and Runtime API support offered for some of these languages may be limited depending on the state of the current implementation.
 
-This page explains the various levels of support for source languages in Reactr, as well as lists the currently supported languages and the potential caveats associated with them.
-
+This page explains the various levels of support for source languages in Reactr. See [Creating Runnables](https://docs.suborbital.dev/atmo/usage/creating-runnables/) for the `--lang` parameters for each.
```

Comment:
I'm not sure why the second half of the sentence got removed here?

Also I would say the remark about creating runnables could go on its own paragraph, I'd preface it with something like 
> "**Our `subo` command line tool can help you scaffold** _(or 'create' if you would like to avoid jargon)_ **starter templates for runnables in the various supported languages,** refer to creating runnables...".

Reply:
> I'm not sure why the second half of the sentence got removed here?

It seemed redundant, like first we're going to tell you the support status of the languages we support, and then give you a list of the same languages and caveats that are the same as their support status. But maybe I missed some nuance?

Reply:
> Also I would say the remark about creating runnables could go on its own paragraph, I'd preface it with something like
> 
> > "**Our `subo` command line tool can help you scaffold** _(or 'create' if you would like to avoid jargon)_ **starter templates for runnables in the various supported languages,** refer to creating runnables...".

👍 I've added this and moved the "This page explains... " sentence to the top of the page.